src/components/settings-row/settings-row.scss

```diff
@@ -31,12 +31,12 @@
   box-sizing: border-box;
   clear: both;
   float: left;
-  width: 25%;
+  width: calc(30% - 10px);
```

Comment:
since you've changed the other value to `65`, should this not be `35`? Perhaps with a larger margin than `10px`?